src/requestLib.js

```diff
@@ -127,10 +128,10 @@ var RequestLib = {
    * @param {http.IncomingMessage} clientRequest
    * @param {Number} retries
    */
-  call: function (params, clientRequest, retries) {
+  call: function (params, clientRequest, requestType, retries) {
```

Comment:
it's more like `server type` instead of request type.

Request Type naming seems confusing...

Reply:
Fixed

Reply:
can be resolved.